package.json

```diff
@@ -112,6 +113,8 @@
     "ts-jest": "^29.1.0",
     "ts-loader": "^9.4.2",
     "ts-node": "^10.9.1",
+    "ts-transform-paths": "^3.0.0",
```

Comment:
I'd remove it from ui build process because it's related to sdk part which will be in a separate repo.